### PR TITLE
Kill containers if they exceed their time limit

### PR DESCRIPTION
**Pull request by eugulixes**

### CODE REVIEW

File: dominion/base.py [hunk 1]
```diff
@@ -18,7 +18,11 @@
 
 from images.models import Image
```
Comment (karpa4o4) [inline dominion/base.py:20]: Unnecessary line splitting.

File: dominion/tasks.py [hunk 2]
```diff
@@ -12,45 +12,63 @@
...
         'name': container_name,
         'environment': env,
     }
-    self.request.kwargs['container'] = DOCKER_CLIENT.containers.run('count-von-count', **run_kwargs)
-    for line in self.request.kwargs['container'].logs(follow=True, stream=True):
+    container = self.request.kwargs['container'] = DOCKER.run('count-von-count', **run_kwargs)
+
+    watch.apply_async((image_id, ), queue=QUEUE_WATCH_NAME)
+
+    channel_name = f'build-log-{image_id}'
```
Comment (karpa4o4) [inline dominion/tasks.py]: `channel_name` pattern might be worth moving to the settings, as `CONTAINER_NAME`.

File: dominion/base.py (hunk 1 above)
Comment (eugulixes) [reply dominion/base.py:20]: The images package is a third-party relatively Dominion, so the blank line is necessary here.

Comment (karpa4o4) [reply dominion/base.py:20]: Then let's add the blank line [here](https://github.com/tolstoyevsky/dominion/pull/25/files#diff-715432cad783bf48cb9358d0aa4c16c04eb3b203edeb08a515a6163128d7d8f7R28).

File: dominion/tasks.py (hunk 2 above)
Comment (eugulixes) [reply dominion/tasks.py:63]: Agreed. But we don't touch it here, so let's move it to the settings when it's really necessary.

File: dominion/tasks.py [hunk 3]
```diff
@@ -12,48 +12,105 @@
...
+    LOGGER.info(f'Watching {container_name}')
+
+    try:
+        container = DOCKER.get(container_name)
+    except NotFound:
+        LOGGER.info(f'{container_name} does not exist, so finishing the task')
+        return
+
+    retries_number = TIMEOUT // POLLING_FREQUENCY
+    for i in range(retries_number):
```
Comment (karpa4o4) [inline dominion/tasks.py]: Let's mark the `i` variable as unused.

File: dominion/base.py (hunk 1 above)
Comment (eugulixes) [reply dominion/base.py:20]: It's done in https://github.com/tolstoyevsky/dominion/pull/26

**Review by karpa4o4 — APPROVED** [review]

**Pull request merged**